dfttopif/parsers/base.py

```diff
@@ -179,137 +194,165 @@ def get_positions(self):
         raise NotImplementedError
 
     def get_U_settings(self):
-        '''Get the DFT+U settings, if used
+        """
+        Get the DFT+U settings, if used
 
         Returns: Value, which could contain several keys
             'Type' -> String, type of DFT+U employed
             'Values' -> dict of Element -> (L, U, J)
         Note: Returns None if DFT+U was not used
-        '''
+        """
 
         raise NotImplementedError
 
     def get_vdW_settings(self):
-        '''Get the vdW settings, if applicable
+        """
+        Get the vdW settings, if applicable
 
-        Returns: Value where `scalars` is the name of the vdW method. None if vdW was not used'''
+        Returns: Value where `scalars` is the name of the vdW method. None if vdW was not used
+        """
 
         raise NotImplementedError
 
     # Operations for retrieving results
     def _is_converged(self):
-        '''Read output to see whether it is converged
+        """
+        Read output to see whether it is converged
         
         Hidden operation: self.is_converged() is the public
         interface, which may draw from a converged result
         
-        Returns: boolean'''
+        Returns: boolean
+        """
         
         raise NotImplementedError
         
     def get_total_energy(self):
-        '''Get the total energy of the last ionic step
+        """
+        Get the total energy of the last ionic step
         
         Returns: Property
-        '''
+        """
         
         raise NotImplementedError
         
     def get_band_gap(self):
-        '''Get the band gap energy
+        """
+        Get the band gap energy
 
-        Returns: Property'''
+        Returns: Property
+        """
 
         raise NotImplementedError
 
     def get_pressure(self):
-        '''Get the pressure acting on the system
+        """Get the pressure acting on the system
 
-        Returns: Property, where pressure is a scalar'''
+        Returns: Property, where pressure is a scalar
+        """
 
         raise NotImplementedError
 
     def get_dos(self):
-        '''Get the total density of states
+        """
+        Get the total density of states
 
-        Returns: Property where DOS is a vector, and the energy at which the DOS was evaluated is a condition'''
+        Returns: Property where DOS is a vector, and the energy at which the DOS was evaluated is a condition
+        """
             
         raise NotImplementedError
 
     def get_stresses(self):
-       '''Get the stress tensor
-
-       Returns: Property where stresses is a 2d matrix'''
+        """
+        Get the stress tensor
 
-       raise NotImplementedError
+        Returns: Property where stresses is a 2d matrix
+        """
+        raise NotImplementedError
 
     def get_total_force(self):
-       return None
+        return None
 
     def get_total_magnetization(self):
-       return None
+        return None
+
+    def get_absorption(self):
```

Comment:
If this only applies to `Wien2k`, why don't we leave it in that parser and overload `get_result_functions` instead.